src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -390,19 +392,19 @@ public MappingsSaver mappingsSaver() {
     @Override
     public Notifier notifier() {
         String notifier = (String) optionSet.valueOf(NOTIFIER);
-        if ("ConsoleNotifier".equals(notifier)) {
+        if (ConsoleNotifier.class.getSimpleName().equals(notifier)) {
             return new ConsoleNotifier(verboseLoggingEnabled());
         }
-        if ("Slf4jNotifier".equals(notifier)) {
+        if (Slf4jNotifier.class.getSimpleName().equals(notifier)) {
             return new Slf4jNotifier(verboseLoggingEnabled());
         }
         else {
             try {
-                ClassLoader loader = Thread.currentThread().getContextClassLoader();
-                Class<?> cls = loader.loadClass(notifier);
-                return (Notifier)cls.newInstance();
+                @SuppressWarnings("unchecked")
+                Class<Notifier> notifierClass = (Class<Notifier>)ClassUtils.getClass(notifier);
+                return ConstructorUtils.invokeConstructor(notifierClass);
             } catch (Exception e) {
```

Comment:
I don't really like the catch all exceptions approach. Do you want to have it more specific?